build.sbt

```diff
@@ -15,10 +15,15 @@ lazy val commonDependencies = Seq(
   Dependencies.Libraries.config,
   Dependencies.Libraries.slf4j,
   Dependencies.Libraries.log4jOverSlf4j,
+  Dependencies.Libraries.log4jApi,
+  Dependencies.Libraries.log4jCore,
```

Comment:
I had to add this to fix the issue with previous build failure: https://travis-ci.org/snowplow/snowplow-elasticsearch-loader/jobs/420217876